src/index.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from "./AmplifyMapLibreGeocoder";
 import { createDefaultIcon } from "./createDefaultIcon";
 import { drawGeofences } from "./drawGeofences";
-import { AmplifyGeofenceControl } from "./AmplifyGeofenceControl";
+import { AmplifyGeofenceControl } from "./AmplifyGeofenceControl/index";
```

Comment:
Roll up did not like leaving the AmplifyGeofenceControl/index.ts file so changed it to this

Reply:
Did you actually need the `/index` at all? Normally it would just grab that file if you are referencing a directory.